cloud: add the cluster pausing status (#14376)
pingcap/docs: tidb-cloud/pause-or-resume-tidb-cluster.md

4. In the dialog, click **Pause** to confirm your choice.

>     After you click **Pause**, the cluster will be in the **Pausing** status first. Once the pause operation is done, the cluster will be in the **Paused** status.

Reviewer: ```suggestion
    After you click **Pause**, the cluster will enter the **Pausing** status first. Once the pause operation is done, the cluster will transition to the **Paused** status.
```
